plugins/woocommerce/tests/e2e-pw/tests/merchant/order-status-filter.spec.js

```diff
@@ -55,26 +56,15 @@ test.describe(
 			await api.post( 'orders/batch', { delete: [ ...orderBatchId ] } );
 		} );
 
-		test( 'should filter by All', async ( { page } ) => {
-			await page.goto( '/wp-admin/admin.php?page=wc-orders' );
-
-			await page.locator( 'li.all > a' ).click();
-			// because tests are running in parallel, we can't know how many orders there
-			// are beyond the ones we created here.
-			for ( let i = 0; i < orderStatus.length; i++ ) {
-				const statusTag = 'text=' + orderStatus[ i ][ 0 ];
-				const countElements = await page.locator( statusTag ).count();
-				await expect( countElements ).toBeGreaterThan( 0 );
-			}
-		} );
-
```

Comment:
This test is the same as the tests in the for loop below. I removed it, and added `[ 'All', 'all' ],` in `orderStatus` so this case is now covered in for loop.